src/SuperimposeMesh/CMakeLists.txt

```diff
@@ -25,6 +25,41 @@ find_package(glm    REQUIRED)
 find_package(OpenCV REQUIRED)
 find_package(OpenGL REQUIRED)
 
+# Workaround for flawed assimp package on Ubuntu: remove non-existing paths (like "/usr/lib/include")
+if(UNIX AND assimp_FOUND AND ASSIMP_INCLUDE_DIRS)
+    message(AUTHOR_WARNING "Calling ASSIMP workaround to remove non-existing paths")
+
+    # Remove non-existing paths
+    foreach(path ${ASSIMP_INCLUDE_DIRS})
+        if(NOT EXISTS ${path})
+            list(REMOVE_ITEM ASSIMP_INCLUDE_DIRS "${path}")
+        endif()
+    endforeach()
+
+    # Check if ASSIMP_INCLUDE_DIRS contains at least one valid path
+    set(VALID_ASSIMP_INCLUDE_DIRS False)
+    foreach(path ${ASSIMP_INCLUDE_DIRS})
+        if(EXISTS "${path}/assimp/ai_assert.h")
+            set(VALID_ASSIMP_INCLUDE_DIRS True)
+        endif()
+    endforeach()
+
+    # Find a valid include directory if necessary
+    if(NOT ${VALID_ASSIMP_INCLUDE_DIRS})
+        find_file(ASSIMP_HEADER_FILE "/assimp/ai_assert.h" PATHS "/usr/include")
```

Comment:
Hi @jchevrie, sorry to come back now on this 😄 
I just want to suggest that instead of using `find_file` we could use `find_path`. It is the exact same command, but hte output is already the directory containing the file. As a result, we will only require a single `get_filename_component` call instead of two and it will be much more easy to understend the code in the future.

Also, could you please add an entry in the [`CHANGELOG.md`](https://github.com/robotology/superimpose-mesh-lib/blob/devel/CHANGELOG.md) file so we can keep track of the changes?

After that I'll merge the PR 👍 
Thanks again for contributing! 🎉